CHANGES.md

```diff
@@ -7,6 +7,7 @@ Next release (5.2.1)
 
 Features
 --------
+* [#1050](https://github.com/java-native-access/jna/pull/1050): Add `c.s.j.p.win32.VersionHelpers` and supporting functions - [@dbwiddis](https://github.com/dbwiddis).
```

Comment:
Please update the version behind "Next release" to "5.3.0", as it will be a feature release with this change.

Reply:
Thought so and somewhat expected this comment. :)